src/main/resources/db/changelog/ddl/0008_EIP1-2833_alter_delivery.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="nathan.russell@valtech.com" id="0008_EIP1-2833_alter_delivery - add addressFormat column" context="ddl">
+
+        <addColumn tableName="delivery">
+            <column name="address_format" type="varchar(20)" value="UK">
```

Comment:
Adds the new column, and sets all existing records to have the value `UK` (thats the only address format we support at the moment)

Reply:
Minor comment, @NathanRussellValtech. 
We have set this column size to 50 in VCA, so it might be good to set it to 50 here to be consistent. 
```

  <changeSet author="abdullah.sanver@valtech.com" id="0072_EIP1-2819_add_certificate_delivery" context="ddl">
        <createTable tableName="certificate_delivery">
            <column name="id" type="uuid">
                <constraints primaryKey="true" nullable="false" primaryKeyName="certificate_delivery_pkey"/>
            </column>
            <column name="voter_card_application_id" type="uuid">
                <constraints references="voter_card_application(id)" foreignKeyName="fk_certificate_delivery_voter_card_application" nullable="false" />
            </column>
            <column name="line1" type="varchar(255)">
                <constraints nullable="false"/>
            </column>
            <column name="line2" type="varchar(255)" />
...
            <column name="address_format" type="varchar(50)">
                <constraints nullable="false"/>
            </column>
            <column name="delivery_address_type" type="varchar(50)">
                <constraints nullable="false"/>
            </column>
```

Reply:
Done; thanks 👍